vue/test/base.js

```diff
@@ -39,6 +42,38 @@ const getComponent = function(
     return wrapper;
 };
 
+/**
+ * Initializes and mount a custom component.
+ *
+ * @param {Object} component Custom component to initialize.
+ * @param {Object} options An options object with:
+ * - 'props' - The props to initialize the component with.
+ * - 'route' - Optionally, define the current route for Vue Router.
+ * - 'mixins' - Optionally, add or override mixins for the component.
+ * - 'mocks' - The set of mocks to apply.
+ * - 'provide' - Optionally, add properties for components to use in injections
+ * - 'store' - The definition of the Vuex store
+ */
+const getCustomComponent = function(

```

Comment:
Can't you just use an existing simple component just to avoid all this?

Reply:
Using an invalid name to use the deault `{}` ?